acceptance-tests/src/test-support/java/tech/pegasys/artemis/test/acceptance/dsl/ArtemisNode.java

```diff
@@ -137,39 +138,70 @@ public UnsignedLong getGenesisTime() throws IOException {
     return fetchGenesisTime();
   }
 
-  public void waitForNewBlock() throws IOException {
-    final Bytes32 startingBlockRoot = getCurrentBeaconHead().getBlockRoot();
-    waitFor(
-        () -> assertThat(getCurrentBeaconHead().getBlockRoot()).isNotEqualTo(startingBlockRoot));
+  public void waitForNewBlock() {
+    final Bytes32 startingBlockRoot = waitForBeaconHead().getBlockRoot();
+    waitFor(() -> assertThat(waitForBeaconHead().getBlockRoot()).isNotEqualTo(startingBlockRoot));
```

Comment:
Probably should use `fetchBeaconHead` here.
```suggestion
    waitFor(() -> assertThat(fetchBeaconHead().getBlockRoot()).isNotEqualTo(startingBlockRoot));
```